docs/src/module.md

```diff
@@ -129,6 +129,28 @@ M = Singular.Module(R, v1, v2, v3)
 G = std(M; complete_reduction=true)
 ```
 
+### Reduction
+
+```@docs
+reduce(::smodule, ::smodule)
+```
+
+**Examples**
+
+```julia
```

Comment:
This PR has no coverage. If you turned this example into a jldoctest, we'd get coverage.